assets/runner_cell/src/App.js

```diff
@@ -189,7 +189,7 @@ function HelpBox({ fields }) {
           target="_blank"
           className="border-b border-gray-900 font-medium text-gray-900 no-underline hover:border-none"
         >
-          FLAME
+          FLAME{" "}
```

Comment:
Let's put it after the tag so the space is not underlined :D

Reply:
Oops, that's right.

Done.
![CleanShot 2024-09-06 at 10 33 03](https://github.com/user-attachments/assets/2cbc73a4-1b0e-41ae-a618-b5ddaffef15f)